ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/parsers/TomlParser.kt

```diff
@@ -118,6 +136,9 @@ public value class TomlParser(private val config: TomlConfig) {
         return this
     }
 
+    private fun String.trimCommentLine() =
```

Comment:
Actually we already have some logic related to trimming of comments. 
But yes, it looks much better here. May be you can remove all usages of: `content.findBeginningOfTheComment` though?

Reply:
I'll replace it with a `takeBeforeComment` function, since it's always used to trim the comment away

Reply:
awesome, totally agree